neqo-transport/src/lib.rs

```diff
@@ -28,7 +28,7 @@ mod quic_datagrams;
 mod recovery;
 mod recv_stream;
 mod rtt;
-mod send_stream;
+pub mod send_stream;
```

Comment:
See how @martinthomson does it here with a new feature: https://github.com/mozilla/neqo/pull/1594/files#diff-ea9df8dd0e492efc27d0690a0529971f8debe8ff01291cee81ca7db8ab1e6465R31-R33